ui/app/__mocks__/react-router-dom.js

```diff
@@ -0,0 +1,6 @@
+const originalModule = jest.requireActual('react-router-dom');
```

Comment:
is this an automock? Does jest look for these in every sub-level folder? I thought it only looked for node_modules mocks at the app root.

Reply:
This behaves like if you would have `"automock": true` in jest config, except it only mocks this node module and only if that module is imported somewhere in the `ui/app` folder. More about mocking node modules in Jest here: https://jestjs.io/docs/manual-mocks#mocking-node-modules

Without this we would need to do jest.mock('react-router-dom') in many React files. With this we just do it once here and then this mock version will be used every time we import that module in our React files:
```import { useHistory } from 'react-router-dom';```

Reply:
cool, i didn't realize it would mock node_modules via relative path like that. neat.